diff --git a/changes/libevent-compat b/changes/libevent-compat
new file mode 100644
index 000000000..e69de29bb
diff --git a/configure.ac b/configure.ac
index 58eab736d..862a2912f 100644
--- a/configure.ac
+++ b/configure.ac
@@ -478,7 +478,8 @@ TOR_SEARCH_LIBRARY(libevent, $trylibeventdir, [-levent $STATIC_LIBEVENT_FLAGS $T
 #endif
 #include <sys/time.h>
 #include <sys/types.h>
-#include <event2/event.h>], [
+#include <event2/event.h>
+#include <event2/event_compat.h>], [
 #ifdef _WIN32
 #include <winsock2.h>
 #endif
